### PR TITLE
Basecamp 3 provider

### PR DESCRIPTION
**Pull request by miralita**
Hello,
here is Basecamp 3 provider
image

### CODE REVIEW

File: src/provider/Basecamp.ts [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    }
+
+    public async parseData(): Promise<void> {
+        // console.log(this.body)
+        // fs.appendFile('bc.json', JSON.stringify(this.body) + '\n\n', function (err) {
+        //     if (err) {
+        //         return console.log(err)
+        //     }
+        // })
+        //let embed: Embed
```
Comment (Jawnnypoo) [inline src/provider/Basecamp.ts:36]: Can you remove these commented out bits please?

**Review by Jawnnypoo — APPROVED** [review]
This is great, thanks for adding this!

**Comment by Jawnnypoo** [comment]
If you have the time, can you also add a test to this with some test JSON?

**Comment by Jawnnypoo** [comment]
I can do the follow ups actually, thanks a bunch!

**Comment by miralita** [comment]
Thanks for the great tool! In fact, I have never code in TypeScript before, but your code is so well-structured and the library has such a good API that it was simple for me to write new provider :)

**Pull request merged**